autoload/vimlparser.vim

```diff
@@ -2875,6 +2881,29 @@ function! s:ExprTokenizer.get_dstring()
   return s
 endfunction
 
+function! s:ExprTokenizer.get_dict_literal_key()
+  call self.reader.skip_white()
+  let r = self.reader
+  let c = r.peek()
+  if !s:isalnum(c) && c != '_' && c != '-'
+    throw s:Err(printf('unexpected token: %s', token.value), token.pos)
```

Comment:
`token` is not defined here.

Reply:
oops, I missed the warning... (because current autoload/vimlparser.vim has lots of vint errors!)
fixed in #126

Reply:
Thanks!

You might want to try this `.vintrc.yaml`:
```
policies:
  ProhibitNoAbortFunction:
    enabled: false
```

(FWIW Neomake includes the policy name in the output format)